src/contracts/rewarder/ODefaultStakerRewards.sol

```diff
@@ -55,10 +66,122 @@ contract ODefaultStakerRewards is
+            $.activeSharesCache[epoch] = $$.activeSharesCache[epoch];
+            $.claimableAdminFee[epoch][tokenAddress] = $$.claimableAdminFee[epoch][tokenAddress];
+            unchecked {
+                ++epoch;
+            }
+        }
+    }
+
+    // function migrateOperatorClaimed(uint48 startEpoch, uint48 endEpoch, address tokenAddress) external {
+    //     PreviousStakerRewardsStorage storage $$ = _getOldStakerRewardsStorage();
+    //     StakerRewardsStorage storage $ = _getStakerRewardsStorage();
+    //     address middleware = INetworkMiddlewareService(i_networkMiddlewareService).middleware(i_network);
+    //     // all on chain => so we read from the previous storage location and we migrate
+    //     for (uint48 epoch = startEpoch; epoch < endEpoch; epoch++) {
+    //         uint256[] memory previousRewards = $$.rewards[epoch][tokenAddress];
+    //         uint48 epochTs = EpochCapture(middleware).getEpochStart(epoch);
+    //         uint256 activeSharesCache = $$.activeSharesCache[epoch];
+    //         address[] memory operators = IOBaseMiddlewareReader(middleware).activeOperatorsAt(epochTs);
+    //         for (uint256 k; k < operators.length;) {
+    //             address account = operators[k];
+
+    //             if ($.stakerClaimedRewardPerEpoch[account][epoch][tokenAddress] != 0) {
+    //                 ++k;
+    //                 continue;
+    //             }
+
+    //             for (uint256 j; j < $$.lastUnclaimedReward[account][epoch][tokenAddress];) {
+    //                 uint256 amount = IVault(i_vault).activeSharesOfAt(account, epochTs, new bytes(0)).mulDiv(
+    //                     previousRewards[j], activeSharesCache
+    //                 );
+
+    //                 $.stakerClaimedRewardPerEpoch[account][epoch][tokenAddress] += amount;
+    //                 unchecked {
+    //                     ++j;
+    //                 }
+    //             }
+    //             unchecked {
+    //                 ++k;
+    //             }
+    //         }
+    //     }
+    // }
+
+    function migrateOperatorClaimed(uint48 startEpoch, uint48 endEpoch, address tokenAddress) external {
+        PreviousStakerRewardsStorage storage $$ = _getOldStakerRewardsStorage();
+        StakerRewardsStorage storage $ = _getStakerRewardsStorage();
+        address middleware = INetworkMiddlewareService(i_networkMiddlewareService).middleware(i_network);
+        // all on chain => so we read from the previous storage location and we migrate
+        for (uint48 epoch = startEpoch; epoch < endEpoch; epoch++) {
+            uint256[] memory previousRewards = $$.rewards[epoch][tokenAddress];
+            uint48 epochTs = EpochCapture(middleware).getEpochStart(epoch);
+            uint256 activeSharesCache = $$.activeSharesCache[epoch];
+            address[] memory operators = IOBaseMiddlewareReader(middleware).activeOperatorsAt(epochTs);
+            for (uint256 k; k < operators.length;) {
```

Comment:
I'm being picky here, but it's weird we are using `k`, it could be regular i instead. I know why we got to `k`, but asking to go back no that there's no `i` and `j` before.